readthedocs/restapi/views/integrations.py

```diff
@@ -84,15 +110,26 @@ class GitHubWebhookView(WebhookMixin, APIView):
         }
     """
 
-    def handle_webhook(self, request, project, data=None):
+    def get_payload(self):
+        if self.request.content_type == 'application/x-www-form-urlencoded':
+            try:
+                return json.loads(self.request.data['payload'])
+            except ValueError:
+                pass
```

Comment:
`return None` would be more explicit here. Also feels weird to have it all nested in an `if` block -- should probably do a test for `!= 'application/x-www-form-urlencoded'` at the top.